src/Integration/VsSessionHost.cs

```diff
@@ -16,15 +16,17 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
 using System.Diagnostics;
+using System.Globalization;
 using System.Linq;
 using System.Windows.Threading;
 
 namespace SonarLint.VisualStudio.Integration
 {
     [Export(typeof(IHost))]
     [PartCreationPolicy(CreationPolicy.Shared)]
-    internal sealed class VsSessionHost : IHost, IProgressStepRunnerWrapper, IDisposable
+    internal sealed class VsSessionHost : IHost, IProgressStepRunnerWrapper, IDisposable, IServiceContainer
```

Comment:
Probably needed to be one once it hosted services, but now is required since I need to replace one of the real service with a test service in tests and I didn't want to expose the dictionary of the services to avoid implementing IServiceContainer which is cleaner IMO.